src/status_im/chat/subs.cljs

```diff
@@ -186,27 +171,30 @@
     (get-in db [:unviewed-messages chat-id :count])))
 
 (reg-sub :web-view-extra-js
-  (fn [db]
-    (let [chat-id (subscribe [:get-current-chat-id])]
-      (get-in db [:web-view-extra-js @chat-id]))))
+  :<- [:get-current-chat]
+  (fn [current-chat]
+    (:web-view-extra-js current-chat)))
 
 (reg-sub :all-messages-loaded?
-  (fn [db]
-    (let [chat-id (subscribe [:get-current-chat-id])]
-      (get-in db [:chats @chat-id :all-loaded?]))))
+  :<- [:get-current-chat]
+  (fn [current-chat]
+    (:all-loaded? current-chat)))
 
 (reg-sub :photo-path
   :<- [:get-contacts]
   (fn [contacts [_ id]]
     (:photo-path (contacts id))))
 
+;; TODO janherich: this is just bad and horribly ineffecient (always sorting to get last msg +
```

Comment:
Can you create an issue to track this?

Reply:
We already have general issue for that - https://app.zenhub.com/workspace/o/status-im/status-react/issues/1875